src/toolkit/matl_buy_policy.h

```diff
@@ -66,6 +66,7 @@ class MatlBuyPolicy : public Trader {
   /// @param manager the agent
   /// @param buf the resource buffer
   /// @param name a unique name identifying this policy
+  /// @param throughput a maximum on total transaction quantities
```

Comment:
Could you expand a bit on this documentation to clarify the purpose of throughput?  I needed to re-read the PR notes to understand the motivation of this addition.